.github/workflows/ngen_integration.yaml

```diff
@@ -84,6 +86,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: noaa-owp/ngen
+          ref: refs/pull/832/head
```

Comment:
Similar to previous comment, is this intended for long term or simply stop gapped for testing the PR?